tests/data/normal_data.py

```diff
@@ -0,0 +1,475 @@
+import geomstats.backend as gs
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.information_geometry.normal import (
+    CenteredNormalDistributions,
+    CenteredNormalMetric,
+    DiagonalNormalDistributions,
+    DiagonalNormalMetric,
+    GeneralNormalDistributions,
+)
+from tests.data_generation import _OpenSetTestData, _RiemannianMetricTestData
+
+
+class CenteredNormalDistributionsTestData(_OpenSetTestData):
+    Space = CenteredNormalDistributions
+    sample_dim_list = [3, 5, 10]
+    space_args_list = [(sample_dim,) for sample_dim in sample_dim_list]
+    shape_list = [(sample_dim, sample_dim) for sample_dim in sample_dim_list]
+    n_samples_list = [1, 3, 5]
+    n_points_list = [1, 3, 5]
+    n_vecs_list = [1, 3, 5]
+
+    def belongs_test_data(self):
+        sample_dim = self.sample_dim_list[0]
+        n_samples = self.n_samples_list[0]
+        random_data = [
+            dict(
+                sample_dim=sample_dim,
+                point=self.Space(sample_dim).random_point(n_samples=n_samples),
+                expected=gs.array([True] * n_samples),
+            )
+        ]
+        sample_dim = self.sample_dim_list[1]
+        n_samples = self.n_samples_list[1]
+        random_data.append(
+            dict(
+                sample_dim=sample_dim,
+                point=self.Space(sample_dim).random_point(n_samples=n_samples),
+                expected=gs.array([True] * n_samples),
+            )
+        )
```

Comment:
This data is "repeated" because `random_point_belongs_test_data` does exactly this.

(same applies for other test data)

Reply:
What change do you suggest here ?

Reply:
I was suggesting to delete all the cases that generate random points, because `random_point_belongs` does exactly that (check line218 of `data_generation.py`). If I'm not missing anything, the "automatic" code generates exactly the same test cases as your code here (the data points will be different only due to randomness).